modules/assim.batch/vignettes/MultiSitePDAVignette.Rmd

```diff
@@ -0,0 +1,646 @@
+    <sigma>0</sigma>
+    <sigma>1</sigma>
+   </quantiles>
+   <variable>NEE</variable>
+   <ensemble.id>1000017325</ensemble.id>
+   <start.year>2007</start.year>
+   <end.year>2014</end.year>
+  </settings.5>
+  <settings.6>
+   <quantiles>
+    <sigma>-1</sigma>
+    <sigma>0</sigma>
+    <sigma>1</sigma>
+   </quantiles>
+   <variable>NEE</variable>
+   <ensemble.id>1000017325</ensemble.id>
+   <start.year>1999</start.year>
+   <end.year>2006</end.year>
+  </settings.6>
+  <settings.7>
+   <quantiles>
+    <sigma>-1</sigma>
+    <sigma>0</sigma>
+    <sigma>1</sigma>
+   </quantiles>
+   <variable>NEE</variable>
+   <ensemble.id>1000017325</ensemble.id>
+   <start.year>2013</start.year>
+   <end.year>2015</end.year>
+  </settings.7>
+  <settings.8>
+   <quantiles>
+    <sigma>-1</sigma>
+    <sigma>0</sigma>
+    <sigma>1</sigma>
+   </quantiles>
+   <variable>NEE</variable>
+   <ensemble.id>1000017325</ensemble.id>
+   <start.year>1997</start.year>
+   <end.year>2010</end.year>
+  </settings.8>
+ </sensitivity.analysis>
+```
+
+## Settings for HB-PDA
+
+Similar to PDA settings, we will contain HB-PDA settings under the `<assim.batch>` section. Likewise standard PDA, if you have chosen the parameters you want to target in the calibration, open the `pecan.CONFIGS.xml` file, insert the tag below, and save as `pecan.HBPDA.xml`:
+
+```
+<assim.batch>
+  <method>emulator.ms</method>
+  <mode>local</mode>
+  <n.knot>60</n.knot>
+  <iter>10000</iter>
+  <chain>3</chain>
+  <param.names>
+   <soil>
+    <param>som_respiration_rate</param>
+    <param>soil_respiration_Q10</param>
+    <param>soilWHC</param>
+   </soil>
+   <temperate.deciduous>
+    <param>half_saturation_PAR</param>
+    <param>dVPDSlope</param>
+    <param>psnTOpt</param>
+   </temperate.deciduous>
+  </param.names>
+</assim.batch>
+```
+
+Different from the PDA settings, this time the main function will be the `emulator.ms`. This function can be run in three modes, which will be selected under the `<mode>` tag above: local, global and hierarchical. If left empty, the function will go through all three modes. But note that, for both global and hierarhical calibration, we need site-level runs. We will start with local mode, i.e. site-level calibration. 
+
+Now we will extend the settings above to include information about data to assimilate and likelihoods, similar to the standard-PDA setup, except for multiple sites. First, add `<assim.batch>` tag to the multisettings in the bottom of your `pecan.HBPDA.xml`:
+
+```
+ <multisettings>
+  <multisettings>assim.batch</multisettings>
+  <multisettings>ensemble</multisettings>
+  <multisettings>sensitivity.analysis</multisettings>
+  <multisettings>run</multisettings>
+ </multisettings>
+```
+
+Now we can insert multi-site settings in the `<assim.batch>` section accordingly (note that now the chunk above where we defined `method`, `mode`, `param.names` etc. will go under each `setting.*`):
+
+```
+  <assim.batch>
+  <settings.1>
```

Comment:
The design below seems to have a lot of redundant information. Inputs are site specific, but why/how would method, mode, n.know, iter, chain, param.names, be different between sites?

Reply:
they wouldn't be different but otherwise I was not able to load `assim.batch` tags as multi settings properly. 

I'm not familiar with multi.settings usage, and couldn't see any documentation. So I was just mimicking what is being written when we write a multi.settings list back to an xml (e.g. the ensemble tag above) to make it work. I can play around a little bit more.

Reply:
Like with the other multisettings (e.g. ensemble), I bet there's a way to get it to do a lot of the expansion automatically. Probably OK for it to be this way in the current PR, but let's try to make things easier for the user by the next release